aerleon/lib/nacaddr.py

```diff
@@ -408,12 +444,14 @@ def CollapseAddrList(addresses, complement_addresses=None):
     )
 
 
-def SortAddrList(addresses):
+def SortAddrList(addresses: List[Union[Any, IPv6, IPv4]]) -> List[Union[Any, IPv6, IPv4]]:
     """Return a sorted list of nacaddr objects."""
     return sorted(addresses, key=ipaddress.get_mixed_type_key)
 
 
-def RemoveAddressFromList(superset, exclude):
+def RemoveAddressFromList(
+    superset: List[Union[IPv4, IPv6]], exclude: Union[IPv4, IPv6]
+) -> List[Union[Any, IPv6, IPv4]]:
```

Comment:
This Any and the one on line 447 seem suspect also.

Reply:
Removing them (if it is correct to do so) will also remove the import for typing.Any